src/api/app/instrumentations/workflow_placeholder_variables_instrumentation.rb

```diff
@@ -7,7 +7,7 @@ def track_placeholder_variables(workflows_file_content)
     return if placeholder_variables.blank?
 
     supported_placeholder_variables = placeholder_variables.select do |placeholder_variable|
-      Workflows::YAMLToWorkflowsService::SUPPORTED_PLACEHOLDER_VARIABLES.include?(placeholder_variable)
+      Workflows::YAMLToWorkflowsService::SUPPORTED_PLACEHOLDER_VARIABLES.include?(placeholder_variable.to_sym)
```

Comment:
We need to convert `placeholder_variable` to a symbol since the Regex captures from line 5 are going to be strings.